inginious/frontend/flask/mongo_sessions.py

```diff
@@ -68,7 +68,7 @@ def _get_signer(self, app):
 
     def open_session(self, app, request):
         # Check for cookieless session in the path
-        path_session = re.match(r"(/@)([a-f0-9A-F_]*)(@)", request.path)
```

Comment:
I tried to look into the code for other instances of this kind of check by searching for "/@" . I think I found all of them.